From 7d9cbd69612e71bbf9fe6c0e08618c0e91af5e15 Mon Sep 17 00:00:00 2001
From: Denis Benato <denis.benato@linux.dev>
Date: Fri, 16 Jan 2026 19:06:36 +0100
Subject: [PATCH 47/49] platform/x86: asus-armoury: keep the list ordered
 alphabetically

Model GA403WM appears after GA403WR breaking the alphabetical order:
swap theirs position.

Fixes: f5fc40734b0f ("platform/x86: asus-armoury: add support for GA403WM")
Signed-off-by: Denis Benato <denis.benato@linux.dev>
---
 drivers/platform/x86/asus-armoury.h | 64 ++++++++++++++---------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/drivers/platform/x86/asus-armoury.h b/drivers/platform/x86/asus-armoury.h
index a79c7887b..6ab92ce15 100644
--- a/drivers/platform/x86/asus-armoury.h
+++ b/drivers/platform/x86/asus-armoury.h
@@ -955,38 +955,6 @@ static const struct dmi_system_id power_limits[] = {
 			.requires_fan_curve = true,
 		},
 	},
-	{
-		.matches = {
-			DMI_MATCH(DMI_BOARD_NAME, "GA403WR"),
-		},
-		.driver_data = &(struct power_data) {
-			.ac_data = &(struct power_limits) {
-				.ppt_pl1_spl_min = 15,
-				.ppt_pl1_spl_max = 80,
-				.ppt_pl2_sppt_min = 25,
-				.ppt_pl2_sppt_max = 80,
-				.ppt_pl3_fppt_min = 35,
-				.ppt_pl3_fppt_max = 80,
-				.nv_dynamic_boost_min = 0,
-				.nv_dynamic_boost_max = 25,
-				.nv_temp_target_min = 75,
-				.nv_temp_target_max = 87,
-				.nv_tgp_min = 80,
-				.nv_tgp_max = 95,
-			},
-			.dc_data = &(struct power_limits) {
-				.ppt_pl1_spl_min = 15,
-				.ppt_pl1_spl_max = 35,
-				.ppt_pl2_sppt_min = 25,
-				.ppt_pl2_sppt_max = 35,
-				.ppt_pl3_fppt_min = 35,
-				.ppt_pl3_fppt_max = 65,
-				.nv_temp_target_min = 75,
-				.nv_temp_target_max = 87,
-			},
-			.requires_fan_curve = true,
-		},
-	},
 	{
 		.matches = {
 			DMI_MATCH(DMI_BOARD_NAME, "GA403WM"),
@@ -1019,6 +987,38 @@ static const struct dmi_system_id power_limits[] = {
 			.requires_fan_curve = true,
 		},
 	},
+	{
+		.matches = {
+			DMI_MATCH(DMI_BOARD_NAME, "GA403WR"),
+		},
+		.driver_data = &(struct power_data) {
+			.ac_data = &(struct power_limits) {
+				.ppt_pl1_spl_min = 15,
+				.ppt_pl1_spl_max = 80,
+				.ppt_pl2_sppt_min = 25,
+				.ppt_pl2_sppt_max = 80,
+				.ppt_pl3_fppt_min = 35,
+				.ppt_pl3_fppt_max = 80,
+				.nv_dynamic_boost_min = 0,
+				.nv_dynamic_boost_max = 25,
+				.nv_temp_target_min = 75,
+				.nv_temp_target_max = 87,
+				.nv_tgp_min = 80,
+				.nv_tgp_max = 95,
+			},
+			.dc_data = &(struct power_limits) {
+				.ppt_pl1_spl_min = 15,
+				.ppt_pl1_spl_max = 35,
+				.ppt_pl2_sppt_min = 25,
+				.ppt_pl2_sppt_max = 35,
+				.ppt_pl3_fppt_min = 35,
+				.ppt_pl3_fppt_max = 65,
+				.nv_temp_target_min = 75,
+				.nv_temp_target_max = 87,
+			},
+			.requires_fan_curve = true,
+		},
+	},
 	{
 		.matches = {
 			DMI_MATCH(DMI_BOARD_NAME, "GA503QR"),
-- 
2.52.0

